.github/workflows/lsp-release.yml

```
@@ -25,10 +25,6 @@ jobs:
         python-version: "3.10"
 
     - run: |
-
-        sudo apt update
-        sudo apt install pandoc
-
         python --version
         python -m pip install --upgrade pip
         python -m pip install --upgrade tox bump2version towncrier docutils
```

```suggestion
        python -m pip install --upgrade tox hatch towncrier docutils
```
Just caught something I missed the other day :)